examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/AuthenticationTest.kt

```diff
@@ -96,28 +88,62 @@ class AuthenticationTest: RealmTest() {
             // :remove-end:
             val user = app.login(Credentials.emailPassword(email, password))
             assertEquals(User.State.LOGGED_IN, user.state) // :remove:
+            user.delete() // :remove:
         }
         // :snippet-end:
+        app.close()
     }
 
     @Test
-    fun deleteUserTest() {
+    fun removeUserTest() {
         val email = getRandom()
         val password = getRandom()
-        // :snippet-start: user-delete
-        val app: App = App.create(YOUR_APP_ID) // Replace this with your App ID
-        runBlocking { // use runBlocking sparingly -- it can delay UI interactions
+        // :snippet-start: remove-user
+        val app = App.create(YOUR_APP_ID) // Replace with your App ID
+        runBlocking {
             // :remove-start:
             app.emailPasswordAuth.registerUser(email, password)
             val credentials = Credentials.emailPassword(email, password)
             // :remove-end:
+            // Log user in
             val user = app.login(credentials)
-            // use the user object ...
+            assertEquals(User.State.LOGGED_IN, user.state) // :remove:
+
+            // Work with logged-in user ...
+
+            // Remove the user from the app
+            // If the user is logged in, they are logged out first
+            // DOES NOT delete user from the server
```

Comment:
To be consistent with our messaging, we should probably change this.
```suggestion
            // DOES NOT delete user from the App Services App
```

Reply:
updated